### ISSUE TITLE
To-do List

### ISSUE THREAD

**Issue by yolandawww** [yolandawww/QASystem#36]
- hyperparameters
    * embedding:100-> 200?
    * overfitting:
        * [x] (fei) q, c first LSTM weights share
        * [x] (junjie) L2 & ema wd
        * dropout=0.6
    * train+dev
    * \ ans without . :(start 0.66%, end 2.57%) 
- model
    - [ ] (fei) add character representation
    - [x] (junjie) DP for sentence
    - [x] (yuanfang) attention similarity
    - [ ] (fei) diff
        * logits = get_logits([g1, p0], with_dropout
        * bi-attention LSTM(end predict)
    - [ ] aggregation layer from multi-perspective?
- embedding
- poster to do
   - [x] (junjie) 画一个bi-att的model图
   - [x] training loss, validation loss (fei)
   - [x] 加不加drop-out (fei)
   - [x] 分析不work的case
   - [ ] robust embedding
- paper
